api/v1alpha1/nodepool_types.go

```diff
@@ -89,7 +89,11 @@ type NodePoolSpec struct {
 type NodePoolStatus struct {
 	// NodeCount is the most recently observed number of replicas.
 	// +optional
```

Comment:
Is this still +optional without the pointer?

Reply:
it was not a pointer, it's just changing from int to int32